docs/learn/clientside-interpolation.md

```diff
@@ -27,75 +27,12 @@ In client-side Interpolation instead of just snapping objects to their positions
 
 Normally a client in a server-authoritative topology, barring any additional tricks and techniques, would be able to render state that is approximately half the Round Trip Time (RTT) behind the actual state of simulation on the server. In order for client-side interpolation to be able to work it needs to be somewhat behind (catching up to) the most recent state passed to us from the server. In effect, our latency would increase by our [Interpolation Period](../reference/glossary/ticks-and-update-rates#interpolation-period). In order to avoid stutter, we want that period to be less than the [Packet Sending Period](../reference/glossary/ticks-and-update-rates#packet-sending-period). When the client is done interpolating to the previous state, it would always have received a new state to repeat the process. 
 
-:::unity Future  Netcode for GameObjects (Netcode) Feature
+Client-side interpolation is implemented in Netcode for GameObjects (Netcode) in the [NetworkTransform](../components/networktransform.md) component.
```

Comment:
as discussed, let's check that NetworkTransform doc talks about needing custom interpolation server side